source/upgrade.txt

```diff
@@ -52,6 +56,27 @@ changes between the current and upgrade versions. For example, if you
 are upgrading the driver from v4.0 to v4.5, address all breaking changes from
 the version after v4.0 including any listed under v4.5.
 
+.. _java-breaking-changes-v5.0:
+
+Version 5.0 Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- The driver removes the following record annotations:
+  
+  - ``BsonId``
+  - ``BsonProperty``
+  - ``BsonRepresentation``
```

Comment:
Was this meant to be included in this PR?

Reply:
yes I added a note in the PR description - I think this step wasnt done when you did the ticket, so just handled it here!